### PR TITLE
cleanup quo2 2

### PR DESCRIPTION
**Pull request by flexsurfer**
cleanup quo2, remove `status-im` namespace usage

create utils root namespace
move preview screens to status-im namespace

### CODE REVIEW

**Review by OmarBasem — APPROVED** [review]
All looks good to me. Just one question, why do we have a `utils.cljs` file outside of utils directory?

**Comment by flexsurfer** [comment]
> All looks good to me. Just one question, why do we have a `utils.cljs` file outside of utils directory?

because it's like general js utils, and they are used in quo as well, but we don't want to use status-im in quo namespace

File: src/quo2/components/buttons/dynamic_button.cljs [hunk 1]
```diff
@@ -93,8 +92,8 @@
                                                 :notification-up   0
                                                 :search-with-label 8)}}
            (case type
-             :jump-to                                       (i18n/label :t/jump-to)
-             :search-with-label                             (i18n/label :t/back)
+             :jump-to                                       label
+             :search-with-label                             label
```
Comment (Parveshdhull) [inline src/quo2/components/buttons/dynamic_button.cljs:96]: Please can you elaborate, why this change required.

File: src/quo2/components/switcher/switcher_cards.cljs [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
             [quo2.components.tags.status-tags :as status-tags]
-            [status-im.ui.components.fast-image :as fast-image]
             [quo2.components.avatars.user-avatar :as user-avatar]
             [quo2.components.avatars.group-avatar :as group-avatar]
             [quo2.components.list-items.preview-list :as preview-list]
-            [quo2.components.avatars.channel-avatar :as channel-avatar]))
+            [quo2.components.avatars.channel-avatar :as channel-avatar]
+            [status-im.i18n.i18n :as i18n]))
+
+;;TODO move outside quo or find a way how to move i18n outside
```
Comment (Parveshdhull) [inline src/quo2/components/switcher/switcher_cards.cljs:16]: moving outside quo2 will break the Figma structure we are following for components?
`quo2/componenet_directory/component`

**Review by Parveshdhull — APPROVED** [review]
apart from above comments, looks good

**Comment by J-Son89** [comment]
> > All looks good to me. Just one question, why do we have a `utils.cljs` file outside of utils directory?
> 
> because it's like general js utils, and they are used in quo as well, but we don't want to use status-im in quo namespace

I think it's beyond the scope of this work but probably at some point we should look to split that utils file up into pieces that are more specific to each ones functionality.

**Comment by J-Son89** [comment]
Great work 😎

**Review by ilmotta — APPROVED** [review]

**Review by ilmotta — COMMENTED** [review]
Thanks for moving forward with these refactorings @flexsurfer

**Review by jo-mut — APPROVED** [review]

File: src/quo2/components/buttons/dynamic_button.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/quo2/components/buttons/dynamic_button.cljs:96]: good question, i wanted quo library as pure as possible, so it shouldn't depend on status translation, so we need to pass labels as parameters to component, but there might be difficulties for sure, but at least we should try, probably there is better implementation for dynamic button, but my goal was just to move i18n outside in this PR

Comment (Parveshdhull) [reply src/quo2/components/buttons/dynamic_button.cljs:96]: cool, sounds good. Thank you
